package.json

```diff
@@ -16,14 +16,17 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node pretest.js",
     "test": "mocha --timeout 20000 --require test/init.js",
-    "posttest": "npm run lint"
+    "posttest": "node posttest.js && npm run lint"
```

Comment:
This won't get run if the tests fail. Perhaps there is something that can be done in some sort of "after all" type hook in mocha instead?